src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
 // write code here
+const lists = document.querySelector('ul');
+
+sortList(lists);
```

Comment:
The `sortList` function is called here, and again on line 43. Calling it twice on the same list is redundant. Consider removing one of these calls.